### ISSUE TITLE
Nice error screen! Oh F:4001,0,20004A94

### ISSUE THREAD

**Issue by morganrallen** [Emeryth/sma-q2-oss#3]
Red background, message:
```
F:4001,0,20004A94
E:0x3
```

revision e4d6f2c

**Comment by Emeryth** [comment]
Something happened.

**Comment by x29a** [comment]
@morganrallen do you get that stacktrace at a specific action, e.g. whenever you connect via bluetooth? im there now :(

**Comment by x29a** [comment]
This could be related to some memory corruption in flash. Does it happen right after firmware boot?

**Comment by morganrallen** [comment]
Yes this happened right after flashing, with no interaction required. As I recall it would just boot loop with that error. I never went further in debugging but am finally setup to start tinkering again.

**Comment by x29a** [comment]
Hey welcome back!

Please see the linked issue #15 and give the latest revision a try (see [this commit](https://github.com/Emeryth/sma-q2-oss/commit/7fd32fd3b44fbb9046013784fdde5417e89d0acc)).
